go.mod

```
@@ -0,0 +1,3 @@
+module hw/courses
```

:memo: За прийнятою конвенцією, модулі в Go називаються так, як репа:
```suggestion
module github.com/senchov/go_hausaufgaben
```

Почитати можна тут https://go.dev/ref/mod#module-path

---

що значить будування?

---

добавив коментарі де викорастано структури nil

---

Під "будуванням" мається на увазі "Struct Embedding" https://gobyexample.com/struct-embedding